pkg/loop/server.go

```diff
@@ -68,16 +69,42 @@ func (s *Server) start() error {
 		return fmt.Errorf("error getting environment configuration: %w", err)
 	}
 
-	if err := SetupTracing(TracingConfig{
+	tracingConfig := TracingConfig{
 		Enabled:         envCfg.TracingEnabled,
 		CollectorTarget: envCfg.TracingCollectorTarget,
 		SamplingRatio:   envCfg.TracingSamplingRatio,
 		TLSCertPath:     envCfg.TracingTLSCertPath,
 		NodeAttributes:  envCfg.TracingAttributes,
 		OnDialError:     func(err error) { s.Logger.Errorw("Failed to dial", "err", err) },
-	}); err != nil {
-		// non blocking to server start
-		s.Logger.Errorf("Failed to setup tracing: %s", err)
+	}
+
+	if envCfg.TelemetryEndpoint == "" {
+		err := SetupTracing(tracingConfig)
+		if err != nil {
+			return fmt.Errorf("failed to setup tracing: %w", err)
+		}
+	} else {
+		beholderCfg := beholder.Config{
+			InsecureConnection:       envCfg.TelemetryInsecureConnection,
+			CACertFile:               envCfg.TelemetryCACertFile,
+			OtelExporterGRPCEndpoint: envCfg.TelemetryEndpoint,
+			ResourceAttributes:       append(tracingConfig.Attributes(), envCfg.TelemetryAttributes.AsStringAttributes()...),
+			TraceSampleRatio:         envCfg.TelemetryTraceSampleRatio,
+		}
+
+		if tracingConfig.Enabled {
+			exporter, err := tracingConfig.NewSpanExporter()
+			if err != nil {
+				return fmt.Errorf("failed to setup tracing exporter: %w", err)
+			}
+			beholderCfg.TraceSpanExporter = exporter
+		}
+
+		beholderClient, err := beholder.NewClient(beholderCfg)
+		if err != nil {
+			return fmt.Errorf("failed to create beholder client: %w", err)
+		}
+		beholder.SetClient(beholderClient)
```

Comment:
@jmank88,  `SetupTracing` sets global otel TracerProvider in the end [here](https://github.com/smartcontractkit/chainlink-common/blob/14a5c7af361f61afb7e02360e286012770edf8f0/pkg/loop/telem.go#L134-L135). 
Its reflected in README in chainlink repo see [here](https://github.com/smartcontractkit/chainlink/blob/72f4cc8aaa128202fd5974c6dbfb29c6beb1be12/.github/tracing/README.md?plain=1#L78) 

To achieve parity in behavior we need to do the same after beholder client is created.
At minimum set global tracer
```
  otel.SetTracerProvider(tracerProvider)
  otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}))
```
or set all globals
 
```
beholder.SetGlobalOtelProviders
```


See `otel.GetTracerProvider` calls in chainlink core:
- https://github.com/smartcontractkit/chainlink/blob/72f4cc8aaa128202fd5974c6dbfb29c6beb1be12/core/services/pg/connection.go#L56
- https://github.com/smartcontractkit/chainlink/blob/72f4cc8aaa128202fd5974c6dbfb29c6beb1be12/core/web/router.go#L66

```suggestion
		beholder.SetClient(beholderClient)
		
		// Set global providers including Tracer to have parity with 'SetupTracing'
		beholder.SetGlobalOtelProviders()
```

Reply:
We can have parity in the tracing systems and how they function, but there's no real/immediate need for it. Tracing will be used and IFIUC gradually phased out internally in support of Beholder. Externally no NOPs are currently using the tracing feature. 

IFIUC the only thing that won't function as intended is if both tracing and beholder are enabled as the globally registered tracing provider could conflict.

Reply:
Updated to call `beholder.SetGlobalOtelProviders`